src/main/js/main_html5.js

```diff
@@ -873,11 +875,37 @@ require("../../../html5-common/js/utils/environment.js");
             lang: audioTracks[index].language,
             enabled: audioTracks[index].enabled
           };
+          if (audioTracks[index].enabled) {
+            currentAudioId = audioTracks[index].id;
+          }
           audioTrackList.push(element);
         }
-        return audioTrackList;
       }
-      return [];
+      return audioTrackList;
+    };
+
+    /**
+     * Sets the audio track to the ID specified by trackID
+     * @public
+     * @method OoyalaVideoWrapper#setAudio
+     * @param {String} trackId - the ID of the audio track to activate
+     * @returns {Boolean} true - if new audio track was set; false is in otherwise;
```

Comment:
Small type, could be just "false otherwise"

Reply:
\+